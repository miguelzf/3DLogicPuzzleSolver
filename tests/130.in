- - - Y - -
- B - - - -
R - - G - -
- - P - - X
- - - - - -
- L - - - X

- G - - - X
- - - - - -
X - - X - -
- - - - - -
- X - - - -
- - - - - -

X - - - - -
- - - - - -
- - - - Y -
L - - - - -
- - P - - -
- - - R - B
