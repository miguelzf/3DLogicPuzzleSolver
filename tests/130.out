
Solution:
-  -  -  Y  y  y  
-  B  b  b  b  b  
R  g  g  G  p  p  
r  g  P  p  p  X  
r  g  g  g  g  g  
r  L  l  l  l  X  
================
r  G  g  -  l  X  ||  X  g  p  p  b  y  
r  r  g  -  l  -  ||  -  g  p  b  b  y  
X  r  g  X  l  -  ||  -  g  p  b  Y  y  
r  r  g  -  l  l  ||  L  g  p  b  -  -  
r  X  g  g  g  g  ||  g  g  P  b  b  b  
r  r  r  r  r  r  ||  r  r  r  R  -  B  
